pkg/models/cedar_authority_to_operate.go

```diff
@@ -0,0 +1,37 @@
+package models
+
+import (
+	"github.com/guregu/null/zero"
+)
+
+// CedarAuthorityToOperate is the model for ATO information that comes back from the CEDAR Core API
+type CedarAuthorityToOperate struct {
+	ActualDispositionDate                     zero.Time `json:"actualDispositionDate"`
+	CedarID                                   string    `json:"cedarId"` // required
+	ContainsPersonallyIdentifiableInformation bool      `json:"containsPersonallyIdentifiableInformation"`
+	CountOfTotalNonPrivilegedUserPopulation   int32     `json:"countOfTotalNonPrivilegedUserPopulation"`
+	CountOfOpenPoams                          int32     `json:"countOfOpenPoams"`
+	CountOfTotalPrivilegedUserPopulation      int32     `json:"countOfTotalPrivilegedUserPopulation"`
+	DateAuthorizationMemoExpires              zero.Time `json:"dateAuthorizationMemoExpires"`
+	DateAuthorizationMemoSigned               zero.Time `json:"dateAuthorizationMemoSigned"`
+	EAuthenticationLevel                      string    `json:"eAuthenticationLevel"`
+	Fips199OverallImpactRating                int32     `json:"fips199OverallImpactRating"`
+	FismaSystemAcronym                        string    `json:"fismaSystemAcronym"`
+	FismaSystemName                           string    `json:"fismaSystemName"`
+	IsAccessedByNonOrganizationalUsers        bool      `json:"isAccessedByNonOrganizationalUsers"`
+	IsPiiLimitedToUserNameAndPass             bool      `json:"isPiiLimitedToUserNameAndPass"`
+	IsProtectedHealthInformation              bool      `json:"isProtectedHealthInformation"`
+	LastActScaDate                            zero.Time `json:"lastActScaDate"`
+	LastAssessmentDate                        zero.Time `json:"lastAssessmentDate"`
+	LastContingencyPlanCompletionDate         zero.Time `json:"lastContingencyPlanCompletionDate"`
+	LastPenTestDate                           zero.Time `json:"lastPenTestDate"`
+	PiaCompletionDate                         zero.Time `json:"piaCompletionDate"`
+	PrimaryCyberRiskAdvisor                   string    `json:"primaryCyberRiskAdvisor"`
+	PrivacySubjectMatterExpert                string    `json:"privacySubjectMatterExpert"`
+	RecoveryPointObjective                    float32   `json:"recoveryPointObjective"`
+	RecoveryTimeObjective                     float32   `json:"recoveryTimeObjective"`
+	SystemOfRecordsNotice                     []string  `json:"systemOfRecordsNotice"`
+	TlcPhase                                  string    `json:"tlcPhase"`
+	XlcPhase                                  string    `json:"xlcPhase"`
```

Comment:
Bit of a nit-pick, but I've been digging into go naming standards, and acronyms are typically best all in the same case (either all-caps, or no-caps). Since these need to be public fields on the struct, I think `TLCPhase` and `XLCPhase` might work a bit better!

Reply:
Looks good as of https://github.com/CMSgov/easi-app/pull/1636/commits/b421c6bbac17cdbb5fb9086448d5bcce549eb523.